jest.config.js

```diff
@@ -0,0 +1,6 @@
+const { jest: jestConfig } = require('kcd-scripts/config')
+
+module.exports = Object.assign(jestConfig, {
+  roots: ['<rootDir>/src', '<rootDir>/test'],
+  testMatch: ['<rootDir>/test/*.js']
+})
```

Comment:
I have moved tests in test folder as it was before

Reply:
This wasn't necessary, but thank you 😃